spec/models/workshop_spec.rb

```diff
@@ -1,5 +1,13 @@
 require 'rails_helper'
 
 RSpec.describe Workshop, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "association" do
+    let (:sponsor) { create :sponsor }
```

Comment:
dont put a space between `let` and the parenthesis. Do it like on the next line!